### ISSUE TITLE
TestCase's without tests should not show up in SUnit Browser

### ISSUE THREAD

**Issue by ericwinger** [ericwinger/Jade#649]
This class should not show up in the sunit browser: 

![image](https://user-images.githubusercontent.com/36212516/69678290-ffebd200-1059-11ea-8a72-a0e2cc5b1740.png)

Note that it is not specified as ``isAbstract`` but has no test cases. 
A test that could be written compares the browser results to ``RwAbstractTest suite tests`` which is empty for this calss ``RwAbstractTest suite tests ==> anOrderedCollection( )``

**Comment by LisaAlmarode** [comment]
Dale and I observed this on Thursday... I was supposed to verify and file an issue... it's due to the optimizations put in for 3.0.88, that packagesWithTests is no longer executing buildSuite.

**Comment by ericwinger** [comment]
Yes, building a suite was very slow. The suggestion is to achieve the same list as the suite but faster. A test can be written to compare Jadeite results with suite results.

**Comment by LisaAlmarode** [comment]
This is still a bug in 3.2.12.
